### ISSUE TITLE
Docker image / pre-built binary

### ISSUE THREAD

**Issue by christophetd** [awslabs/coldsnap#211]
It would be awesome to provide a Docker image or pre-built binaries! I installed coldsnap manually, but since I don't use  Rust it took me a solid 30 minutes to have everything running.

**Comment by jpculp** [comment]
Hi @christophetd, thanks for reaching out! Let me bring this up to the rest of the team to see what our options are.

**Comment by christophetd** [comment]
Amazing! I really think it would make it much easier to be used more widely and make it more popular. The main alternative right now is `dsnap`, which is easier to run (Python). But if we have a Docker image / pre-built binaries, coldsnap becomes actually easier to use than dsnap.

**Comment by webern** [comment]
My quick take on this is that binaries are probably easier to both create and use than docker images. Most Rust binaries are static and don't require any library linking at runtime, so it's usually as easy as downloading and using the executable binary without any install step required.

**Comment by christophetd** [comment]
If we have a binary it should be relatively trivial adding it to a distroless image as well?

**Comment by webern** [comment]
> If we have a binary it should be relatively trivial adding it to a distroless image as well?

Trivial enough that I don't think we would be excited about publishing it that way (since it's the publication part that isn't so trivial for us).

**Comment by webern** [comment]
> If we have a binary it should be relatively trivial adding it to a distroless image as well?

Yes, in fact it would be so trivial that I don't think publishing the resultant container would confer much benefit.

**Comment by freefly42** [comment]
It would because there are a lot of dependencies to make it work.  For someone who wants to use it temporarily docker makes it really easy to get all the dependencies, and then throw them all away in the end.
